invenio_oauthclient/alembic/97bbc733896c_create_oauthclient_tables.py

```diff
@@ -73,6 +73,7 @@ def upgrade():
         sa.Column('secret', sa.Text(), nullable=False),
         sa.ForeignKeyConstraint(
             ['id_remote_account'], [u'oauthclient_remoteaccount.id'],
+            name='fk_oauthclient_remote_token_remote_account'
```

Comment:
That's just for MySQL. It works on Postgres, so you want to make sure that you change the name for Postgres users too. Otherwise it might backfire in the future...